### PR TITLE
Add program outline

### PR DESCRIPTION
**Pull request by RobJN**
Hopefully this (my html) is all okay.

### CODE REVIEW

**Comment by RobJN** [comment]
@mikelmaron or @tatsvc : Can you check this for poor HTML please (as noted, I think it's fine but this is not my area of expertise) and merge.

File: program/index.html [hunk 1]
```diff
@@ -52,6 +61,7 @@ <h2 class='color-purple2 spartan'>Program</h2>
         </header>
 
         <!-- Friday -->
+        <h2>Friday</h2>
```
Comment (tatsvc) [inline program/index.html]: Can you remove this title? It's repetitive and breaking the layout

image

**Comment by tatsvc** [comment]
Other than the one comment above, this looks good!

**Comment by RobJN** [comment]
@tatsvc We include titles for Saturday and Sunday so wanted to add Friday. I see the formatting is not right here though. Could I move it up a bit to directly after the line "We've got an amazing program of talks"? Would that work?

**Comment by tatsvc** [comment]
Hmm, I honestly think it's fine like this, the sotmus page does the same: http://stateofthemap.us/program/

**Comment by RobJN** [comment]
OK. I've dropped the title as per your recommendation. Thanks :+1:

**Comment by mikelmaron** [comment]
@RobJN looks good to me! only correction needed is to change the data on Monday. I'd do that myself, but for some reason GitHub isn't letting me edit your PR.

Once you make that change, flag me again here, and I can merge the PR.

**Pull request merged**